src/SDKs/Automation/Automation.Tests/Automation.Tests.csproj

```diff
@@ -1,96 +1,38 @@
 ﻿<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
   <PropertyGroup>
-    <Description>Test Project for Automation tests</Description>
-    <VersionPrefix>2.0.0-preview</VersionPrefix>
-    <AssemblyName>Automation.Tests</AssemblyName>
     <PackageId>Automation.Tests</PackageId>
-    <!--<GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>-->
-    <PackageTags>Microsoft AutoRest ClientRuntime REST</PackageTags>
+    <AssemblyName>Automation.Tests</AssemblyName>
+    <VersionPrefix>3.0.0-preview</VersionPrefix>
+    <Description>Test Project for Automation tests</Description>
   </PropertyGroup>
+  
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
     <TestProjectType>true</TestProjectType>
   </PropertyGroup>
-
+  
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-    <PackageReference Include="Microsoft.Azure.Management.Automation" Version="2.0.3" />
-    
-    <PackageReference Include="Hyak.Common" Version="1.1.3" />
-    <PackageReference Include="Microsoft.Azure.Common " Version="2.1.3" />
-    <PackageReference Include="Microsoft.Azure.ResourceManager" Version="1.1.0-preview" />
-  </ItemGroup>
-
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Net" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Net.Http.WebRequest" />
+    <Compile Remove="Customizations\**" />
```

Comment:
@vrdmr remove this item group completely

Reply:
Made the change in the new commit. Pushed in.